### PR TITLE
Bump version of scikit-learn

### PR DESCRIPTION
**Pull request by AlexanderDokuchaev**
### Changes

Bump version of scikit-learn

### Reason for changes

144479

### CODE REVIEW

**Comment by AlexanderDokuchaev** [comment]
manual/job/post_training_quantization/403/

**Comment by AlexanderDokuchaev** [comment]
Trivy_Scan/149

File: tests/onnx/requirements.txt [hunk 1]
```diff
@@ -11,5 +11,5 @@ torch
 torchvision
 
 fastdownload==0.0.7
-scikit-learn==1.2.2
```
Comment (MaximProshin) [inline tests/onnx/requirements.txt:14]: limit it by 1.2.2 from below

Comment (AlexanderDokuchaev) [reply tests/onnx/requirements.txt:14]: done

**Review by MaximProshin — APPROVED** [review]

**Review by alexsu52 — APPROVED** [review]

**Pull request merged**